WORKSPACE

```diff
@@ -38,48 +39,27 @@ gazelle_dependencies()
 # ================================================================
 
 
-http_archive(
+git_repository(
```

Comment:
Should we use the `github_archive` paradigm here as well?

Reply:
it seems like `github_archive` is just light wrapper that doesn't add that much functionality. i'm not anchored one way or another

Reply:
can do as follow-up if requested

Reply:
Not needed, thanks